packages/server/email/provider/google/googleDriver.js

```diff
@@ -0,0 +1,28 @@
+import nodemailer from 'nodemailer'
+import {getGoogleApiConfig} from './getGoogleConfig'
+import {getGlobalConfig} from './../getGlobalConfig'
+import sendToSentry from '../../../utils/sendToSentry'
+
+const config = getGoogleApiConfig()
+const mailGoogle = (getGlobalConfig().provider == 'google')? new nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+         user: config.user,
+         pass: config.pass
+     }
+ }): undefined
+
+if (mailGoogle) {
+  mailGoogle.sendEmail = function (mailOptions) {
```

Comment:
I like what you're doing here!
In this case, I might refactor this into an  `abstract class MailProvider` that is implemented by a `MailProviderGoogle` and `MailProviderMailgun`. That way the `sendMail` contract is clear and if/when we need to add another provider it'll be clear what's needed

Reply:
I will look at it, Thanks

Reply:
Thanks for your feedbak. I will try to do it.